docs/canvas/canvas-workpad.asciidoc

```diff
@@ -69,25 +69,68 @@ Each of the sample data sets comes with a Canvas workpad that you can use for yo
 
 . Add a {kibana-ref}/add-sample-data.html[sample data set].
 
-. On the *Add Data to Kibana* page, click the *View data* dropdown list, then select *Canvas*.
```

Comment:
Add Data to Kibana > Add data

click the *view data* dropdown list > click *View data*